IFC_oM/Settings/IfcSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.ComponentModel;
+
+namespace BH.oM.Adapters.IFC
+{
+    [Description("Settings of the IFC adapter.")]
+    public class IfcSettings
```

Comment:
I'm still confused as to why we have this file when it is empty?

Reply:
True that it could be deleted.

Reply:
It is to be populated in the future (rather near than far) - it will contain stuff like tolerances, IFC schema settings, possibly xBIM parser settings (my gut feeling says there will be quite a few once I learn how to use it) etc. So I prefer adding it straight away to have the core set up from the beginning and avoid versioning soon (especially that the class contains a constructor, so versioning is likely to be a bit more tricky than for usual objects).

Reply:
Ha, I saw the comment of @alelom after posting my response. We are 2 against 1 here - taken my explanation above, would you be OK with keeping it as is? I prefer adding all necessary infrastructure from the beginning instead of changing core things too often.

Reply:
I actually figured out that the reason you left it there empty was the one you stated. I don't mind keeping it. I think by general policy we avoid empty objects always but for Toolkit prototypes at initial stages of dev for me is fine. Last word to Fraser.